src/langcheck/augment/en/_rephrase.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+import os
+from typing import Optional
+
+from openai import AzureOpenAI, OpenAI
+
+
+def rephrase(
+        instances: list[str] | str,
+        *,
+        num_perturbations: int = 1,
+        model_type: str = 'openai',
+        openai_client: Optional[OpenAI] = None,
+        openai_args: Optional[dict[str, str]] = None) -> list[Optional[str]]:
+    '''Rephrases each string in instances (usually a list of prompts) without
+    changing their meaning. We use a modified version of the prompt presented
+    in `"Rethinking Benchmark and Contamination for Language Models with
+    Rephrased Samples" <https://arxiv.org/abs/2311.04850>`__ to make an LLM
+    rephrase the given text.
+
+    We currently support two model types:
+
+    1. The 'openai' type, where we use OpenAI's 'gpt-turbo-3.5' model
+    by default. See `this page
+    <https://langcheck.readthedocs.io/en/latest/metrics.html#computing-metrics-with-openai-models>`__
+    for examples on setting up the OpenAI API key.
+
+    2. The 'azure_openai' type. Essentially the same as the 'openai' type,
+    except that it uses the AzureOpenAI client. Note that you must specify the
+    model to use in `openai_args`, e.g.
+    `openai_args={'model': 'YOUR_DEPLOYMENT_NAME'}`
```

Comment:
Single backticks get rendered as italic, and double backticks as inline code blocks in the documentation. We fixed this on Yosuke's PR too

```suggestion
    model to use in ``openai_args``, e.g.
    ``openai_args={'model': 'YOUR_DEPLOYMENT_NAME'}``
```